lib/Dialect/Handshake/HandshakeOps.cpp

```diff
@@ -96,9 +125,13 @@ static void printOp(OpAsmPrinter &p, Operation *op, bool explicitSize) {
     int size = op->getAttrOfType<IntegerAttr>("size").getValue().getZExtValue();
     p << " [" << size << "]";
   }
+  if (isa<BufferOp>(op)) {
```

Comment:
I think it would be better to just have a specialization for the printer of BufferOp now that we've diverged it's representation from the remainder of the operations.

Reply:
Thanks for your review @mortbopet .  
I will change the code in ```BufferOp::parse() ,BufferOp::print()```. 
And try not to call the function ```sost::parseOperation(), sost::printOp()``` in BufferOp.